back/engines/commercial/multi_tenancy/db/seeds/projects.rb

```diff
@@ -76,12 +76,16 @@ def configure_timeline_for(project)
             description_multiloc: runner.create_for_tenant_locales { Faker::Lorem.paragraphs.map { |p| "<p>#{p}</p>" }.join },
             start_at: start_at,
             end_at: (start_at += rand(150).days),
-            participation_method: %w[ideation budgeting poll information ideation ideation][rand(6)]
+            participation_method: %w[ideation voting poll information ideation ideation][rand(6)]
           })
-          if phase.budgeting?
+          if phase.voting?
             if has_budgeting
```

Comment:
Not entirely sure what this logic does, but seems odd that the else clause is what assigns the budgeting attributes